cli/src/commands/shared_recovery/create.rs

```diff
@@ -84,6 +85,16 @@ pub async fn main(shamir_setup: Args) -> anyhow::Result<()> {
             .collect()
     };
 
+    let threshold = if let Some(t) = threshold {
+        t
+    } else {
+        Input::<NonZeroU8>::new()
+        .with_prompt(format!(
+            "Choose a threshold between 1 and {}\nThe threshold is the minimum number of recipients that one must gather to recover the account",
+            per_recipient_shares.len()
+        )) .interact_text()?
```

Comment:
This is a blocking sync call in an async function, we should at least acknowledge it with a comment